packages/nushell/crates-nu-cli-Cargo.toml.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/crates/nu-cli/Cargo.toml b/crates/nu-cli/Cargo.toml
+index 6368b7b8..720bd6b6 100644
+--- a/crates/nu-cli/Cargo.toml
++++ b/crates/nu-cli/Cargo.toml
+@@ -24,7 +24,7 @@ app_dirs = "1.2.1"
+ async-stream = "0.2"
+ base64 = "0.12.0"
+ bigdecimal = { version = "0.1.0", features = ["serde"] }
+-bson = { version = "0.14.1", features = ["decimal128"] }
```

Comment:
This feature depends on the `decimal` package, whose latest release doesn't contain a fix that allows it to build on non-x86_64 arches, alkis/decimal#52, so disable this feature here and in the next patch.

Reply:
Byte-unit = "6.0.1.0"
Bytes = " 0.5.6"